File: Software/clients/perl_client/src/client_demo.pl
#!/usr/bin/perl -w

use strict;
use AllSparkCubeClient;

my $host = 'localhost';
my $port = 12345;

my $index = 0;
my @data = (255, 0, 255, 0, 255, 0);  # Purple, Green

my $client = AllSparkCubeClient->new($host, $port);
$client->set_data($index, \@data);

print "Success\n"

File: Software/clients/python_client/src/client_demo.py
#!/Usr/bin/env python

from random import randint

from all_spark_cube_client import CubeClient


HOST='192.168.0.100'
PORT=12345

index = 0
data = [randint(0, 255) for x in range(4096 * 3)]

client = CubeClient(HOST, PORT)
client.set_data(index, data)
